enable/tests/primitives/test_image.py

```diff
@@ -110,15 +113,17 @@ def test_image_gc_32(self):
         image_gc = self.image_32._image
         assert_array_equal(image_gc.bmp_array, self.data)
 
+    @unittest.skipIf(six.PY3, reason="Crashes on python 3. See GH #95.")
     def test_draw_24(self):
-        gc = GraphicsContext((256, 128), pix_format='rgba32')
+        gc = GraphicsContext((256, 128), pix_format='rgb24')
```

Comment:
The pix_format is probably wrong. This is a test_draw_24 and it is using rgba32. We might want to add another test to cover the normal case as well?